code/modules/jobs/job_types/vault.dm

```diff
@@ -12,8 +12,6 @@ here's a tip, go search DEFINES/access.dm
 	department_flag = VAULT
 	exp_type = EXP_TYPE_VAULT
 	faction = FACTION_VAULT
-	maptype = "vault"
```

Comment:
Set maptype to Sunnyvale. Already done in a fix PR. Irrelevant.

Reply:
Forgot to set this to Sunnyvale when debugging why Vault roles weren't showing up.